### DISCUSSION TITLE
Query regarding draw.io enterprise version

### DISCUSSION THREAD

**Discussion by Anandhi-23** [jgraph/drawio-desktop#1841] [General]
We are heavily using the free online draw.io editor to create workflow diagrams and other visuals. We are also using Spotify’s Backstage TechDocs platform for our documentation.

We are planning to use the enterprise version of draw.io but haven’t been able to find pricing details. Could you please provide us with information?

We are also integrated draw.io with Viscose using the unofficial extension provided by Henning Dieterichs. Could you provide us with details about the security of this extension and whether it is safe to use?

**Comment by davidjgraph** [comment]
there's no "enterprise" version of diagrams.net, it's a free product.

We're not the authors of the VCCode extension. That said, Henning works for Microsoft's VSCode team, we trust him personally, but we haven't reviewed his code.

**Comment by Anandhi-23** [comment]
Thanks for the input. 

That being said, we only pay for integrating draw.io with Atlassian products.

Has anyone encountered any success integrating 
 draw.io with Spotify’s Backstage? I’ve done some research and it seems there isn’t a plugin available yet.
